compiler/rustc_codegen_llvm/src/coverageinfo/ffi.rs

```diff
@@ -138,7 +138,7 @@ pub(crate) struct CoverageSpan {
 
 impl CoverageSpan {
     pub(crate) fn from_source_region(file_id: u32, code_region: &SourceRegion) -> Self {
-        let &SourceRegion { file_name: _, start_line, start_col, end_line, end_col } = code_region;
+        let &SourceRegion { start_line, start_col, end_line, end_col } = code_region;
```

Comment:
Question: can this `file_id` be `GlobalFileId` or `LocalFileId`?

Reply:
Sure, that's easy enough.